dal/src/grpc/project.proto

```
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package project;
```

Buf (1.54.0)

Files with package "project" must be within a directory "project" relative to root but were in directory "dal/src/grpc".

(PACKAGE_DIRECTORY_MATCH)